app/controllers/shipit/api/locks_controller.rb

```diff
@@ -5,21 +5,23 @@ class LocksController < BaseController
 
       params do
         requires :reason, String, presence: true
+        accepts :lock_level, String
```

Comment:
You can validates the strings directly here, this would generate better API errors.

Reply:
Your gem is awesome !